### ISSUE TITLE
Posts should have ALT Text for accessibility

### ISSUE THREAD

**Issue by jaydeflix** [rjindael/ca-dmv-bot#8]
Or the text of the license plate should be part of the post. As it stands now, the posts are not accessible.

**Comment by jaydeflix** [comment]
Okay. Looks like it's just the web client not showing me alt text.
